app/src/main/java/com/coveros/coverosmobileapp/blogpost/BlogPostsListActivity.java

```diff
@@ -198,6 +222,22 @@ public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCoun
 
             }
         }
+
+    }
+    private class DrawerItemClickListener implements ListView.OnItemClickListener{
+        public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+            switch(position) {
+                case 0:
+                    Intent a = new Intent(getApplicationContext(), MainActivity.class);
```

Comment:
rewrite as `startActivity(new Intent(getApplicationContext(), MainActivity.class));`

use the same style for the one below